api/client/commands.go

```diff
@@ -222,6 +251,9 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 	if *pull {
 		v.Set("pull", "1")
 	}
+
+	v.Set("f", *dockerfileName)
```

Comment:
Since we have the opportunity now before this merges and gets set in stone, shouldn't we give this API parameter a more verbose/meaningful name like `dockerfile`? :angel: